plugins/janus_streaming.c

```diff
@@ -140,6 +140,14 @@ rtsp_user = RTSP authorization username, if needed
 rtsp_pwd = RTSP authorization password, if needed
 rtsp_failcheck = whether an error should be returned if connecting to the RTSP server fails (default=true)
 rtspiface = network interface IP address or device name to listen on when receiving RTSP streams
+rtsp_reconnect_delay = after n seconds passed and no media assumed, the RTSP server has gone and schedule a reconnect (default=5s)
+rtsp_session_timeout = by default the streaming plugin will check the RTSP connection with an OPTIONS query,
+	the value of the timeout comes from the RTSP session initializer and by default
+	this session timeout is the half of this value In some cases this value can be too high (for example more than one minute)
+	because of the media server. In that case this plugin will calculate the timeout with this
+	formula: timeout = min(session_timeout, rtsp_session_timeout / 2). (default=0s)
+rtsp_timeout = communication timeout (CURLOPT_TIMEOUT) for cURL call gathering the RTSP information (default=10s)
+rtsp_conn_timeout = connection timeout for cURL (CURLOPT_CONNECTTIMEOUT) call gathering the RTSP information (default=5s)
```

Comment:
Can you use the same description you used here in the config file too? The one in the config file seems too verbose, and with too many empty lines, and so doesn't mix well with the other values. This would ensure consistency as well (I think in all plugins we have the syntax in config file and doxygen match).

Reply:
Of course, I've synced the source and the config file.